uaclient/status.py

```diff
@@ -1,14 +1,53 @@
+import enum
+
+
 class TxtColor:
     OKGREEN = '\033[92m'
     DISABLEGREY = '\033[37m'
     FAIL = '\033[91m'
     ENDC = '\033[0m'
 
 
+@enum.unique
+class ApplicationStatus(enum.Enum):
```

Comment:
I'm missing something. What is the benefit of this implementation versus the following?

class ApplicationStatus(enum.Enum):
    """
    An enum to represent the current application status of an entitlement
    """
    ENABLED = ACTIVE
    DISABLED = INACTIVE


It looks like we could drop the to_operational_status method by moving this enum definition below the ACTIVE/INACTIVE global declarations.

    https://paste.ubuntu.com/p/f5ZnssXHJ7/

Reply:
alternately, we could drop ACTIVE/INACTIVE altogether right and just us the 'active'/'inactive' strings here couldn't we?

Reply:
My intent is to remove `operational_status()` entirely once this work is complete (replaced by use of the new/renamed axes directly in its current call sites). I expect to remove `to_operational_status()` at the same time.

I used a method instead of the enum values directly here because I wanted to have a clean interface that I could just remove along with `operational_status()`. If I find that it makes sense in the next tranche of work to have ENABLED/DISABLED have useful values (and I expect it will!) then I'll certainly add them, but I didn't want to use them just to support the method I plan to remove in the future.